Marlin/src/lcd/extui/lib/anycubic_chiron/chiron_tft.cpp

```diff
@@ -574,10 +576,11 @@ namespace Anycubic {
       } break;
 
       case 15:   // A15 Resuming from outage
-        if (printer_state == AC_printer_resuming_from_power_outage)
+        if (printer_state == AC_printer_resuming_from_power_outage) {
           // Need to home here to restore the Z position
-          injectCommands_P(AC_cmnd_power_loss_recovery);
-          injectCommands_P(PSTR("M1000"));  // home and start recovery
+          injectCommands(AC_cmnd_power_loss_recovery);
+          injectCommands("M1000");  // home and start recovery
+        }
```

Comment:
It is unclear from the existing indentation whether or not the `M1000` is intended to be conditional. I added braces _presuming_ that both were supposed to be inside the `if` statement.

If this is incorrect, the indentation will need to be fixed and the braces removed.

In the existing code `M1000` would have always replaced the earlier inserted command, since injectCommands_P stores a single string pointer. Switching to `injectCommands` should allow two commands to be injected, although I am not an expert in the proper usage of these commands. I also don't know if this display is ever used on AVR, and whether passing a PSTR to `injectCommands` might be a problem.